prusti-encoder/src/encoders/mir_pure.rs

```diff
@@ -87,7 +78,7 @@ impl TaskEncoder for MirPureEnc {
         Option<DefId>,            // Caller/Use DefID
     );
 
-    type OutputFullLocal<'vir> = MirPureEncOutput<'vir>;
+    type OutputFullDependency<'vir> = MirPureEncOutput<'vir>;
```

Comment:
I'm not sure I understand the motivation for this one. The outputs are generally only used once, right?

Reply:
See comment above